vmdb/app/models/user.rb

```diff
@@ -915,6 +915,11 @@ def self.current_userid=(userid)
     Thread.current[:userid] = userid
   end
 
+  def self.current_user=(user)
+    Thread.current[:user]   = user
+    Thread.current[:userid] = user.try(:userid)
+  end
```

Comment:
Oh wait, we have .current_user just a few lines below...so are we not using it?  I'm just confused on this part :sweat_smile: